packages/alfa-rules/test/sia-r5/rule.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+
+import R5, { Outcomes } from "../../src/sia-r5/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test(`evaluate() passes an html element with lang attribute whose value is a valid primary language subtag`, async (t) => {
+    const target = <html lang="en"></html>;
```

Comment:
Technically, this is not the target, so it should have another name. `html` is likely a good name.
```suggestion
    const html = <html lang="en"></html>;
```
The name change must be propagated.
The same change should be done on all tests.